tests/test_sklearn_metrics.py

```diff
@@ -0,0 +1,231 @@
+import pytest
+import numpy as np
+from omigami.models import sklearn_metrics
+
+
+regression_true = np.array([1, 2, 3, 4, 5])
+regression_pred = np.array([5, 4, 3, 2, 1])
+classification_true = np.array([1, 0, 1, 2, 0, 0])
+classification_pred = np.array([2, 0, 1, 1, 0, 1])
+binary_classification_true = np.array([1, 0, 1, 1, 0, 0])
+binary_classification_pred = np.array([1, 0, 1, 0, 1, 1])
+
+
+@pytest.mark.parametrize(
+    ("y_pred", "y_true", "res"),
+    [(regression_pred, regression_true, -3), (regression_true, regression_true, 1),],
+)
+def test_explained_variance(y_pred, y_true, res):
+    metric = sklearn_metrics.SKLEARN_METRICS["explained_variance"]
+    score = metric(y_true, y_pred)
+    assert res == score
+
+
+@pytest.mark.parametrize(
+    ("y_pred", "y_true", "res"),
+    [(regression_pred, regression_true, -3), (regression_true, regression_true, 1),],
+)
+def test_r2(y_pred, y_true, res):
+    metric = sklearn_metrics.SKLEARN_METRICS["r2"]
+    score = metric(y_true, y_pred)
+    assert res == score
+
+
+@pytest.mark.parametrize(
+    ("y_pred", "y_true", "res"),
+    [(regression_pred, regression_true, -4), (regression_true, regression_true, 0),],
+)
+def test_max_error(y_pred, y_true, res):
+    metric = sklearn_metrics.SKLEARN_METRICS["max_error"]
+    score = metric(y_true, y_pred)
+    assert res == score
+
+
+@pytest.mark.parametrize(
+    ("y_pred", "y_true", "res"),
+    [(regression_pred, regression_true, -2), (regression_true, regression_true, 0),],
+)
+def test_neg_median_absolute_error(y_pred, y_true, res):
+    metric = sklearn_metrics.SKLEARN_METRICS["neg_median_absolute_error"]
+    score = metric(y_true, y_pred)
+    assert res == score
+
+
+@pytest.mark.parametrize(
+    ("y_pred", "y_true", "res"),
+    [
+        (regression_pred, regression_true, -12 / 5),
+        (regression_true, regression_true, 0),
+    ],
+)
+def test_neg_mean_absolute_error(y_pred, y_true, res):
+    metric = sklearn_metrics.SKLEARN_METRICS["neg_mean_absolute_error"]
+    score = metric(y_true, y_pred)
+    assert res == score
+
+
+@pytest.mark.parametrize(
+    ("y_pred", "y_true", "res"),
+    [(regression_pred, regression_true, -8), (regression_true, regression_true, 0),],
+)
+def test_neg_mean_squared_error(y_pred, y_true, res):
+    metric = sklearn_metrics.SKLEARN_METRICS["neg_mean_squared_error"]
+    score = metric(y_true, y_pred)
+    assert res == score
+
+
+@pytest.mark.parametrize(
+    ("y_pred", "y_true", "res"),
+    [
+        (regression_pred, regression_true, -0.587156711483398),
```

Comment:
can't you set res here like you did in the test below, calling a numpy operation?

Reply:
:+1: I will, it's just a complex expression I thought it was clearer to put the number :)